plugins/inventory/kubevirt.py

```diff
@@ -695,36 +567,192 @@ def get_ssh_services_for_namespace(self, client: K8SClient, namespace: str) -> D
 
         return services
 
+    def add_from_namespace(
```

Comment:
maybe `add_hosts_and_groups_from_namespace`? The func name is not very descriptive, IMHO.

Reply:
or `add_to_inventory_from_namespace`?

Reply:
Changed to `populate_inventory_from_namespace`. WDYT?

Reply:
Sounds great!